recipes/ndcube/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
```

Comment:
Please look at using jinja template variables as in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L6